### PR TITLE
ci(ut-controller): codecov report for java

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
codecov report for java
fork repo test screen shot:
image

## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [x] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Comment by tianweidut** [comment]
@anda-ren  update https://github.com/star-whale/starwhale/blob/main/README.md badge ?

File: .github/workflows/server-ut-report.yml [hunk 1]
```diff
@@ -56,4 +56,13 @@ jobs:
...
-        run: make ci-package-with-unit-test-report
\ No newline at end of file
+        run: make ci-package-with-unit-test-report
+
+      - name: Publish coverage to codecov
+        uses: codecov/codecov-action@v3
+        with:
+          name: controller-ut
+          fail_ci_if_error: true
+          flags: controller
```
Comment (tianweidut) [inline .github/workflows/server-ut-report.yml:66]: `flags: controller` > `flags: controller,java` ?

**Comment by anda-ren** [comment]
> @anda-ren update https://github.com/star-whale/starwhale/blob/main/README.md badge ?

@tianweidut done

File: README.md [hunk 2]
```diff
@@ -36,8 +36,8 @@
     <img src="https://github.com/star-whale/starwhale/actions/workflows/console.yml/badge.svg">
 </a>
 
-<a href='https://coveralls.io/github/star-whale/starwhale?branch=main'>
-    <img alt="Coveralls" src="https://img.shields.io/coveralls/github/star-whale/starwhale?label=Java%20cov">
+<a href='https://app.codecov.io/gh/star-whale/starwhale'>
```
Comment (tianweidut) [inline README.md:39]: "https://img.shields.io/codecov/c/github/star-whale/starwhale?flag=controller" ?

File: README.md [hunk 3]
```diff
@@ -36,8 +36,8 @@
...
+    <img alt="Codecov" src="https://img.shields.io/coveralls/github/star-whale/starwhale?flag=controller&label=Java%20cov">
```
Comment (tianweidut) [inline README.md:40]: "https://img.shields.io/codecov/c/github/star-whale/starwhale?flag=controller" ?

Comment (tianweidut) [reply README.md:40]: https://shields.io/category/coverage
image

**Pull request closed without merging**